net/release-notes/release-notes-2022/aspose-pdf-for-net-22-1-release-notes/_index.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Aspose.PDF for .NET 22.1 Release Notes
+linktitle: Aspose.PDF for .NET 22.1
+type: docs
+weight: 120
+url: /net/aspose-pdf-for-net-22-1-release-notes/
+lastmod: "2021-01-17"
+description: This page contains new Aspose.PDF for .NET features, enhancement, and bug fixes in 2022, version 22.1.
+sitemap:
+    changefreq: "weekly"
+    priority: 0.7
+---
+
+{{% alert color="primary" %}}
+
+This page contains release notes information for Aspose.PDF for .NET 22.1.
+
+{{% /alert %}}
+
+## Improvements and Changes
+
+|**Key**|**Summary**|**Category**|
+| :- | :- | :- |
+|PDFNET-50525|PDF2.0 Add saving document support|New Feature|
+|PDFNET-50523|PDF2.0 Add document loading support|New Feature|
```

Comment:
I think we should uniform these titles:
1) Add document `action` support.
or 
2) PDF 2.0 Document `action` support

Reply:
It's just preliminary support of PDF2.0 support (load/save simple files). Full standard support will be available during 2022.